### DISCUSSION TITLE
Has x6 been added to elsa in 2.10.0-rc2 ?

### DISCUSSION THREAD

**Discussion by helloxubo** [elsa-workflows/elsa-core#3633] [Q&A] [answered]
Has x6 been added to elsa in 2.10.0-rc2 ?

**Comment by sfmskywalker** [comment, marked as answer]
Indeed it has. To enable it, add the `use-x6-graphs="true"` attribute to the `<elsa-studio-root`>.

**Reply by helloxubo** [reply]
Thinks.
